packages/edit/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@tailor-cms/tcee-custom-element",
+  "description": "Content element edit template for Vue",
+  "author": "ExtensionEngine <info@extensionengine.com>",
+  "version": "0.0.1",
+  "type": "module",
+  "files": [
+    "dist"
+  ],
+  "main": "./dist/content-element-edit.umd.cjs",
+  "module": "./dist/content-element-edit.js",
+  "exports": {
+    ".": {
+      "import": "./dist/content-element-edit.js",
+      "require": "./dist/content-element-edit.umd.cjs"
+    }
+  },
+  "scripts": {
+    "build": "vue-tsc --noEmit && vite build",
+    "lint": "eslint --ext .js,.ts,.vue ."
+  },
+  "peerDependencies": {
+    "vue": "^2.7.0"
+  },
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^5.31.0",
+    "@typescript-eslint/parser": "^5.31.0",
+    "@vitejs/plugin-vue2": "^2.0.0",
+    "@vue/eslint-config-standard": "^8.0.1",
+    "eslint": "^8.20.0",
+    "eslint-config-prettier": "^8.5.0",
```

Comment:
It might be a good time to replace it with our externalized eslint config.

Reply:
Discussed offline, won't be necessary right now.